desktop-ui/input/hotkeys.cpp

```diff
@@ -139,6 +139,13 @@ auto InputManager::createHotkeys() -> void {
     if(!emulator) return;
     if(settings.audio.volume >= (f64)(0.1)) settings.audio.volume -= (f64)(0.1);
   }));
+
+#if defined(PLATFORM_WINDOWS)  
+  hotkeys.append(InputHotkey("Fork Borderless Window").onPress([&] {
```

Comment:
question: could there be a better label for this than "Fork"? I think for the average user it is not immediately clear what "Fork" means in this case